packages/esm-patient-medications-app/src/routes.json

```diff
@@ -41,7 +41,7 @@
   "workspaces": [
     {
       "name": "add-drug-order",
-      "title": "addDrugOrderWorkspaceTitle",
+      "title": "Add drug order",
```

Comment:
The `title` is the translation key used for mapping to appropriate translation. The correct solution for this will be: The workspace system renders the title as is, instead it should take the translated value as per the `key` and the module the key is coming from: [here](https://github.com/openmrs/openmrs-esm-core/blob/9400555c7c8e50d58ede75cb00059ecbbcbf60f1/packages/framework/esm-styleguide/src/workspaces/workspaces.ts#L621) and [here](https://github.com/openmrs/openmrs-esm-core/blob/9400555c7c8e50d58ede75cb00059ecbbcbf60f1/packages/framework/esm-styleguide/src/workspaces/workspaces.ts#L603)

The approach will be `translateFrom(workspaceModuleName, title)`.

Try to figure out how you can get the `workspaceModuleName`, else you can message me and I'll help out.